tasks/fetch_pod_list.py

```diff
@@ -11,7 +11,8 @@
 
 def fetch_pod_lists():
     logging.info("Fetching pod list")
-    response = requests.get(config.POD_LIST_JSON)
+    response = requests.get(config.POD_LIST_JSON,
+                headers={ "User-Agent" : config.USER_AGENT })
```

Comment:
Minor style issue - I prefer the PEP-8 recommendations here, ie no extra whitespace inside braces, so this should be `headers={"User-Agent": config.USER_AGENT})` - also a few other places where this could be tidied. Just keeps the style consistent. Should write a project style guideline :)